src/components/smart-overflow.cjsx

```diff
@@ -52,7 +53,7 @@ SmartOverflow = React.createClass
   render: ->
     {className} = @props
 
-    classes = "#{className} smart-overflow"
+    classes = classnames 'openstax-smart-overflow', className
 
     <div className={classes} style={@state.style}>
       {@props.children}
```

Comment:
Totally not something you did here, but something I've been gradually leaning towards is inline use of classnames when it's just a few classes because it keeps the logic closer to where it's used  So this could be:

`
     <div className={classnames('openstax-smart-overflow', className)} style={@state.style}>
`